cli/cmd/simulation/quickexact.hpp

```diff
@@ -39,6 +39,8 @@ class quickexact_command : public command
         add_option("--mu_minus,-m", physical_params.mu_minus, "Energy transition level (0/-) (unit: eV)", true);
         add_option("--base,-b", physical_params.base,
                    "2-state (neutral/negative) vs. 3-state (positive/neutral/negative) simulation", true);
+        add_option("--base_number_detection, -bd", params.base_number_detection,
```

Comment:
@Drewniok shorthand mnemonics can only be single characters and cannot be separated via a space. See the other options for examples.

Reply:
Btw, I omitted this option on purpose as I was thinking about omitting `base` as well. It makes sense to always have it set to `ON`, no?

Reply:
@marcelwa Yes, sorry! You are right. This option is not that critical for the CLI.